### ISSUE TITLE
Performance data explorer below acceptable if many categorical fields are specified

### ISSUE THREAD

**Issue by bartcharbon** [molgenis/molgenis#1667]
When for e.g. the lifelines catalogue is loaded, performance drops because the drop down is filled with a great many of lookup tables (the entities ending on "-LUT") for the categorical values.

**Comment by dennishendriksen** [comment]
Yes, solution is to replace the dropdown with the EntitySelectBox React component which only loads more drop down entries on scroll.

**Comment by Mark-de-Haan** [comment]
Is this still an issue? I personally do not notice it at all..

**Comment by dennishendriksen** [comment]
@mark-de-haan Yes, this is still an issue
